imports/ui/components/account-tasks/account-tasks.js

```diff
@@ -47,6 +47,14 @@ Template.accountTasks.events({
 
 Template.loginFrame.onCreated(function() {
 	this.busy(false);
+	this.loginActions = {
+		'Save course': mf('loginAction.saveCourse', 'Login and save course'),
+		'Save event': mf('loginAction.saveEvent', 'Login and save event'),
+		'Save group': mf('loginAction.saveGroup', 'Login and save group'),
+		'Save venue': mf('loginAction.saveVenue', 'Login and save venue'),
+		'Enroll': mf('loginAction.enroll', 'Login and enroll')
+	};
```

Comment:
Ah nice! That way there won't be confusion about what happens when the button is pressed.

Why not do the translation in the calling code? `SaveAfterLogin(instance, mf('loginAction.saveCourse', 'Login and save course'), ...);` that would avoid this table. The only downside I see with this is that the translation won't react to locale changes.

Reply:
Done :) Merging time!